Build error on bench update

hi @deepeshgarg007 and @nabinhait 

Getting the below build error after the latest bench update . Most probably from #25831 or 095048125318814f2c0c36cf49942b7b90879f67


```
$ node esbuild --production --skip_frappe --run-build-command
clean: postcss.plugin was deprecated. Migration guide:
https://evilmartians.com/chronicles/postcss-8-plugin-migration
clean: postcss.plugin was deprecated. Migration guide:
https://evilmartians.com/chronicles/postcss-8-plugin-migration
clean: postcss.plugin was deprecated. Migration guide:
https://evilmartians.com/chronicles/postcss-8-plugin-migration
clean: postcss.plugin was deprecated. Migration guide:
https://evilmartians.com/chronicles/postcss-8-plugin-migration
 > ../erpnext/erpnext/public/js/payment/payments.js:25:27: error: Expected ";" but found ":"
    25 │   set_payment_primary_action: function() {
       ╵                             ^

 > ../erpnext/erpnext/public/js/controllers/taxes_and_totals.js:271:20: error: Expected ";" but found ":"
    271 │   update_item_tax_map: function() {
        ╵                      ^

 > ../erpnext/erpnext/public/js/controllers/transaction.js:1921:24: error: Expected ";" but found ":"
    1921 │   make_quality_inspection: function () {
         ╵                          ^

 ERROR  There were some problems during build

Error: Build failed with 3 errors:
../erpnext/erpnext/public/js/controllers/taxes_and_totals.js:271:20: error: Expected ";" but found ":"
../erpnext/erpnext/public/js/controllers/transaction.js:1921:24: error: Expected ";" but found ":"
../erpnext/erpnext/public/js/payment/payments.js:25:27: error: Expected ";" but found ":"
    at failureErrorWithLog (/home/frappe/frappe-bench/apps/frappe/node_modules/esbuild/lib/main.js:1443:15)
    at /home/frappe/frappe-bench/apps/frappe/node_modules/esbuild/lib/main.js:1125:28
    at runOnEndCallbacks (/home/frappe/frappe-bench/apps/frappe/node_modules/esbuild/lib/main.js:915:63)
    at buildResponseToResult (/home/frappe/frappe-bench/apps/frappe/node_modules/esbuild/lib/main.js:1123:7)
    at /home/frappe/frappe-bench/apps/frappe/node_modules/esbuild/lib/main.js:1230:14
    at /home/frappe/frappe-bench/apps/frappe/node_modules/esbuild/lib/main.js:606:9
    at handleIncomingPacket (/home/frappe/frappe-bench/apps/frappe/node_modules/esbuild/lib/main.js:703:9)
    at Socket.readFromStdout (/home/frappe/frappe-bench/apps/frappe/node_modules/esbuild/lib/main.js:573:7)
    at Socket.emit (events.js:375:28)
    at addChunk (internal/streams/readable.js:290:12)
Done in 5.50s.
Compiling Python files...
________________________________________________________________________________
Bench: Deployment tool for Frappe and Frappe Applications (https://frappe.io/bench).
Open source depends on your contributions, so do give back by submitting bug reports, patches and fixes and be a part of the community :)
```

--- a/erpnext/public/js/controllers/taxes_and_totals.js
+++ b/erpnext/public/js/controllers/taxes_and_totals.js
@@ -268,7 +268,7 @@ erpnext.taxes_and_totals = class TaxesAndTotals extends erpnext.payments {
 		frappe.model.round_floats_in(this.frm.doc, ["total", "base_total", "net_total", "base_net_total"]);
 	}
 
-	update_item_tax_map: function() {
+	update_item_tax_map() {
 		let me = this;
 		let item_codes = [];
 		let item_rates = {};
@@ -305,9 +305,9 @@ erpnext.taxes_and_totals = class TaxesAndTotals extends erpnext.payments {
 				}
 			});
 		}
-	},
+	}
 
-	add_taxes_from_item_tax_template: function(item_tax_map) {
+	add_taxes_from_item_tax_template(item_tax_map) {
 		let me = this;
 
 		if (item_tax_map && cint(frappe.defaults.get_default("add_taxes_from_item_tax_template"))) {
@@ -325,7 +325,7 @@ erpnext.taxes_and_totals = class TaxesAndTotals extends erpnext.payments {
 				}
 			});
 		}
-	},
+	}
 
 	calculate_taxes() {
 		var me = this;

--- a/erpnext/public/js/controllers/transaction.js
+++ b/erpnext/public/js/controllers/transaction.js
@@ -1918,7 +1918,7 @@ erpnext.TransactionController = class TransactionController extends erpnext.taxe
 		});
 	}
 
-	make_quality_inspection: function () {
+	make_quality_inspection() {
 		let data = [];
 		const fields = [
 			{
@@ -2040,9 +2040,9 @@ erpnext.TransactionController = class TransactionController extends erpnext.taxe
 		} else {
 			dialog.show();
 		}
-	},
+	}
 
-	get_method_for_payment: function(){
+	get_method_for_payment() {
 		var method = "erpnext.accounts.doctype.payment_entry.payment_entry.get_payment_entry";
 		if(cur_frm.doc.__onload && cur_frm.doc.__onload.make_payment_via_journal_entry){
 			if(in_list(['Sales Invoice', 'Purchase Invoice'],  cur_frm.doc.doctype)){

--- a/erpnext/public/js/payment/payments.js
+++ b/erpnext/public/js/payment/payments.js
@@ -22,7 +22,7 @@ erpnext.payments = class payments extends erpnext.stock.StockController {
 		});
 	}
 
-	set_payment_primary_action: function() {
+	set_payment_primary_action() {
 		var me = this;
 
 		this.dialog.set_primary_action(__("Submit"), function() {
@@ -107,7 +107,7 @@ erpnext.payments = class payments extends erpnext.stock.StockController {
 		this.bind_numeric_keys_event();
 	}
 
-	bind_form_control_event(){
+	bind_form_control_event() {
 		var me = this;
 		$(this.$body).find('.pos-payment-row').click(function() {
 			me.idx = $(this).attr("idx");
@@ -127,7 +127,7 @@ erpnext.payments = class payments extends erpnext.stock.StockController {
 		$(this.$body).find('.change_amount').change(function() {
 			me.change_amount(flt($(this).val()), precision("change_amount"));
 		});
-	},
+	}
 
 	highlight_selected_row() {
 		var selected_row = $(this.$body).find(repl(".pos-payment-row[idx='%(idx)s']", {'idx': this.idx}));
@@ -135,9 +135,9 @@ erpnext.payments = class payments extends erpnext.stock.StockController {
 		selected_row.addClass('selected-payment-mode');
 		$(this.$body).find('.amount').attr('disabled', true);
 		this.selected_mode.attr('disabled', false);
-	},
+	}
 
-	bind_numeric_keys_event: function() {
+	bind_numeric_keys_event() {
 		var me = this;
 		$(this.$body).find('.pos-keyboard-key').click(function(){
 			me.payment_val += $(this).text();
@@ -192,7 +192,7 @@ erpnext.payments = class payments extends erpnext.stock.StockController {
 		this.frm.doc.change_amount = flt(change_amount, precision("change_amount"));
 		this.calculate_write_off_amount();
 		this.show_amounts();
-	},
+	}
 
 	update_paid_amount(update_write_off) {
 		var me = this;

